course_catalog/models.py

```diff
@@ -504,6 +504,9 @@ class Podcast(LearningResource, LearningResourceGenericRelationsMixin):
     url = models.URLField(null=True, max_length=2048)
     searchable = models.BooleanField(default=True)
 
+    def __str__(self):
```

Comment:
Can we do this or something similar for `PodcastEpisode` too/

Reply:
I was going to do so, but then I though that what I would want to do is something like

```
Episode {self.episode_number} of {self.podcast.title}
```

which would require pulling in the podcast object - what would you suggest for a good `__str__` for this object?

Reply:
I'm fine either way. I don't think the performance hit will be significant since `__str__` is only used in the console for this